pkg/gofr/datasource/file/ftp/go.mod

```diff
@@ -1,6 +1,8 @@
 module gofr.dev/pkg/gofr/datasource/file/ftp
 
-go 1.22
+go 1.22.3
+
+toolchain go1.23.1
```

Comment:
Can we revert this change in this PR as it is out of the scope of the PR.

Reply:
not sure how that happened..